spec/models/user_spec.rb

```diff
@@ -1,5 +1,21 @@
 require 'rails_helper'
 
-RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+# RSpec.describe User, type: :model do
+#   pending "add some examples to (or delete) #{__FILE__}"
+# end
+feature 'determine if user is an admin or not an admin', %Q{
+  An admin is a different role than a member} do
+
+
+    pending  "#admin?" do
```

Comment:
if you do not have admin functionality yet, you should remove these tests from this branch/PR and save them for later.

if you DO, un-mark these as pending please!